pkg/config/store_test.go

```diff
@@ -150,6 +154,10 @@ func TestParse(t *testing.T) {
 				Transparency: TransparencyConfig{
 					URL: "https://rekor.sigstore.dev",
 				},
+				SPIRE: SPIREConfig{
+					Enabled:    false,
+					SocketPath: "/spiffe-workload-api/spire-agent.sock",
```

Comment:
nit: if enabled is false let's remove this, if not it may hint that the path is still somehow necessary

Reply:
on a side note, can we do one enabled with spireMock flag like we did in pipelines? https://github.com/tektoncd/pipeline/pull/4759/files#diff-c199494aad61a99b439167d951412a7ca76efb20d52715d95e6796ae437a335eR33-R36

Reply:
Added the enabled flag. Didnt need to mock it out.

Reply:
Using the mocked spire client for the format_test.go
```
	spireMockClient := &spire.MockClient{}
	var (
		cc spire.ControllerAPIClient = spireMockClient
	)
```